target/linux/ath79/dts/ar9344_mikrotik_sxt5n.dtsi

```diff
@@ -0,0 +1,193 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9344.dtsi"
+
+/ {
+	compatible = "mikrotik,sxt5n", "qca,ar9344";
+	model = "MikroTik SXT5N platform";
+
+	aliases {
+		label-mac-device = &eth1;
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+		serial0 = &uart;
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "sxt5n:green:power";
```

Comment:
we started to use "mikrotik" here for the first part, let's just stick to that (instead of sxt5n) ...